### DISCUSSION TITLE
Multiple Wallet Syncing

### DISCUSSION THREAD

**Discussion by LouisLoudogTrottier** [Chia-Network/chia-blockchain#8384] [Q&A] [answered]
I have about ten wallet syncing for over a week now. Many of then are still at 0 and only the last one is synced.

Whenever the last one get synced (up to height), all wallet stay stuck at the height they were.
If i restart the wallet shile the last wallet is synced, the 1st one start to sync until the lat one get 'out of sync'
... Then the last one sync again and every wallet stop syncing until i restart the wallet again.
If i restart the wallet while the last one is not synced, only the last one sync up and the 9 other don't move at all.

I has been over a week now, and five are still at 0, on is synced and the 4 others are stuck ar 142957,1216,534,7168 respectively.

Anyone know how to fix this?
and NO, Rebboting the machine is NOT a solution.
Restarting the service is NOT a solution just like turning your car on and off while driving is NOT a fix.

**Comment by emlowe** [comment, marked as answer]
chia only syncs one wallet at a time. So only the "active" wallet will sync. You can switch wallets using `chia wallet show` - If using the UI you need to go to the "Keys" tab and select a different key for that wallet to start syncing.

**Reply by LouisLoudogTrottier** [reply]
If i understand correctly, by using ` chia wallet show` and then selecting a wallet in the list will make this wallet the 'active' one? 

I assume it will also make a wallet 'active' by using chia wallet show -f [fingerprint_of_the_wallet_to_make_active] ? is this correct?

**Reply by emlowe** [reply]
Yes

**Reply by LouisLoudogTrottier** [reply]
Awsome, this is great information to know, regards.
